docs/INSTALLATION.md

```diff
@@ -1,13 +1,15 @@
 ## Dart
-* [Install Dart](https://www.dartlang.org/install)
-* Add the Dart SDK to your path.
-* From a new command window, type `dart --version` to verify that Dart is now available.
+* One option is to follow the [instructions for installation on the Dart website](https://www.dartlang.org/install).
```

Comment:
So the section title is 'Dart'? I don't understand.

Reply:
I also suggest adding a table of contents.

Reply:
Agreed, I suggest changing the title to "Installing Dart" or "Recommended installation instructions"

Reply:
@fwip That's what I have done locally. Will be addressing this feedback soon.